libs/client-sdk/src/flag_manager/flag_persistence.cpp

```diff
@@ -21,6 +23,17 @@ static std::string MakeEnvironment(std::string const& prefix,
     return prefix + "_" + PersistenceEncodeKey(sdk_key);
 }
 
+std::unordered_map<std::string, ItemDescriptor> remove_nulls(
```

Comment:
nit: Not a big fan of this being duplicated.

Reply:
Forgot about this. Will fix.